### PR TITLE
Fix and improve v-slot/slot

### PR DESCRIPTION
**Pull request by kooriookami**
Replace all `v-slot` to `#default`, `v-slot:xxx` to `#xxx`, `slot="xxx"` to `#xxx`.

* [x] Make sure you follow Element's contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
* [x] Make sure you are merging your commits to `dev` branch.
* [x] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

**Comment by kooriookami** [comment]
Sorry, I've submitted the last pr as well.

File: website/docs/jp/calendar.md [hunk 1]
```diff
@@ -26,10 +26,9 @@
 :::demo `scoped-slot` に `dateCell` という名前を設定することで、calendarセルに表示する内容をカスタマイズすることができる。`scoped-slot`では、日付(現在のセルの日付)とデータ(type, isSelected, day属性を含む)を取得することができます。詳細は以下のAPIドキュメントを参照のこと。
 ```html
 <el-calendar>
-  <!-- Use 2.5 slot syntax. If you use Vue 2.6, please use new slot syntax-->
   <template
-    slot="dateCell"
-    slot-scope="{date, data}">
+    #dateCell="{data}"
```
Comment (JeremyWuuuuu) [inline website/docs/jp/calendar.md:30]: Seems you lost `date`

Comment (kooriookami) [reply website/docs/jp/calendar.md:30]: I compared with other language docs. They all have no `date`. Is the correct `#dateCell="{date, data}"`?

Comment (zazzaz) [reply website/docs/jp/calendar.md:30]: we cloud just ignore the `date`

**Review by zazzaz — APPROVED** [review]

**Pull request merged**